doc/bisect.rst

```diff
@@ -0,0 +1,87 @@
+*************************
+Code coverage with bisect
+*************************
+
+Normally, preprocessors (specified in ``dune`` files) are constant across all
+builds. However, it usually doesn't make sense to analyze code coverage on all
+builds, especially if you aren't running tests every time.
```

Comment:
I would swap this paragraph and the bellow one, which feels like a more gentle introduction.

Also, the point about preprocessors feels like an implementation detail to me. Instead, we could precise that setting up bisect_ppx in a dune project doesn't create a hard-dependency on bisect_ppx, which is what matters the most to users.